src/sensor/ping360.h

```diff
@@ -63,7 +63,7 @@ class Ping360 : public PingSensor
 
         int nextPoint = (_angle+delta)%_angularResolutionGrad;
 
-        ping360_transducer msg;
+        static ping360_transducer msg;
```

Comment:
This is not good design if we ever instantiate multiple sensor classes. https://stackoverflow.com/a/6223371

You should move the 'ping360_transducer msg' to be a Ping360 class member variable, and then you can also call set_mode(1) only once in the constructor (one more little optimize).

Reply:
Done.